### PR TITLE
Add changedetection.io

### PR DESCRIPTION
**Pull request by steveiliop56**

### CODE REVIEW

File: apps/changedetection/metadata/logo.jpg
Comment (meienberger) [inline apps/changedetection/metadata/logo.jpg:1]: Can you center the icon? It looks weird in the app store

**Review by meienberger — CHANGES_REQUESTED** [review]

File: apps/changedetection/metadata/logo.jpg
Comment (steveiliop56) [reply apps/changedetection/metadata/logo.jpg:1]: Umm its centered... Thats the icon from their repo. (sorry for late response)

Comment (meienberger) [reply apps/changedetection/metadata/logo.jpg:1]: Where in the repo?

Comment (steveiliop56) [reply apps/changedetection/metadata/logo.jpg:1]: Cool

**Review by meienberger — APPROVED** [review]

**Pull request merged**